### DISCUSSION TITLE
zoom setting on q3536

### DISCUSSION THREAD

**Discussion by brehm** [AxisCommunications/discussions#246] [VAPIX] [answered]
Hi 

I upgrade a camera to Q3536, the API used on the previous camera to set the zoom no longer works:

     /axis-cgi/opticssetup.cgi?azoom=N

I cannot find the new equivalent API call, does anyone know what it is or where I could find that.

Regards
/brahima

**Comment by onie5** [comment, marked as answer]
Hi Brahima,

You can check out the Pan/tilt/zoom API or Optics Control API in the VAPIX Library.
